app/src/androidTest/java/com/github/sdpcoachme/schedule/ScheduleActivityTest.kt

```diff
@@ -201,4 +163,20 @@ class ScheduleActivityTest {
             }
         }
     }
+
+    @Test
+    fun dashboardHasRightTitleOnSchedule() {
+        ActivityScenario.launch<ScheduleActivity>(defaultIntent).use {
+            composeTestRule.onNodeWithTag(BAR_TITLE).assertExists().assertIsDisplayed()
+            composeTestRule.onNodeWithTag(BAR_TITLE).assert(hasText("Schedule"))
```

Comment:
It would be better to use assertTrue from JUnit as more info is printed in case the check fails.

Reply:
The assert function comes along with the compose UI tests library and is usefull here to call the hasText(..) matcher. As we work with the compose SemanticsNode here, it is more natural to use these functions.